### DISCUSSION TITLE
Any plans to support JSON Schema validation output format?

### DISCUSSION THREAD

**Discussion by jugaadi** [python-jsonschema/jsonschema#939] [Q&A] [answered]
We use custom vocabulary and therefore, have a need to extract those values based on the correct validation. If the [standard validation output format](https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.12) is supported, we can make use of it to get the validation path so that the keyword value is correctly extracted. This is especially useful in case of compositional/conditional schemas.

**Comment by Julian** [comment, marked as answer]
Yes is the short answer to this, it's definitely still planned. It's a bit nontrivial which is why it's taken so long, but it will definitely come.
